src/feedspora/feedspora_runner.py

```diff
@@ -372,25 +372,29 @@ def post(self, entry):
         maxlen = self._max_len - len(
             post_url) - adjust_with_inner_links - 1  # for last ' '
 
-        # The content with all HTML stripped will be used later, but get it now
-        stripped_html = lxml.html.fromstring(
-            entry.content).text_content().strip()
+        stripped_html = None
+        if entry.content:
+            # The content with all HTML stripped will be used later,
+            # but get it now
+            stripped_html = lxml.html.fromstring(
+                entry.content).text_content().strip()
 
         # Derive additional keywords (tags) from the end of content
         all_keywords = []
-        tag_pattern = r'\s+#([\w]+)$'
-        m = re.search(tag_pattern, stripped_html)
-        while m:
-            tag = m.group(1)
-            if tag not in all_keywords:
-                all_keywords.insert(0, tag)
-            stripped_html = re.sub(tag_pattern, '', stripped_html)
+        if (stripped_html):
```

Comment:
Just quickly browsing the code, these `()` are unnecessary and should be removed :)